osu.Game/Overlays/DashboardOverlay.cs

```diff
@@ -141,6 +159,13 @@ private void onTabChanged(ValueChangedEvent<DashboardOverlayTabs> tab)
 
         private void onlineStateChanged(ValueChangedEvent<APIState> state) => Schedule(() =>
         {
+            if (state.NewValue == APIState.Online)
+                placeholderContainer.Hide();
+            else if (state.NewValue == APIState.Offline)
+            {
+                placeholderContainer.Show();
+                loading.Hide();
```

Comment:
Kind of suspicious how the loading layer is hidden but then not shown again. Did you test that this behaves correctly after the user logs out and then back in again?

Reply:
There is another place in the page code, where the loading spinner is displayed when needed.